project.mml

```diff
@@ -18,7 +18,7 @@ srs: "+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0.0
 
 # Various parts to be included later on
 _parts:
-  # Extents are used for tilemill, and don't actually make it to the generated XML
+  # Extents are used for tilemill, and dont actually make it to the generated XML
```

Comment:
Typo, should be "don't" in English.

Reply:
yeah but that ' break my color syntax :/